js/app/src/lite/Playground.svelte

```diff
@@ -60,37 +60,55 @@
 	});
 
 	function shortcut_run(e: KeyboardEvent): void {
-		if (e.key === "Enter" && (e.metaKey || e.ctrlKey)) {
-			e.preventDefault();
+		if (e.key == "Enter" && (e.metaKey || e.ctrlKey)) {
 			dispatch("code", { code });
+			e.preventDefault();
 		}
 	}
 
+	window.addEventListener("keydown", shortcut_run, true);
```

Comment:
I think this event listener should be attached to each `<Code />` element, instead of the global window.

With the current setting, it behaves like below and I think it's not intuitive.
* `CMD+Enter` triggers a new run even if the editor is not focused.
* When there are multiple Playground elements, a single `CMD+Enter` triggers a new run on all the Playgrounds.